lib/ReadOnlyMiddleware.ts

```diff
@@ -0,0 +1,16 @@
+import { Request, Response, NextFunction } from 'express';
+
+const MONGOKU_READ_ONLY_MODE = process.env.MONGOKU_READ_ONLY_MODE === 'true';
+
+export function readOnlyMode(_: Request, res: Response, next: NextFunction) {
```

Comment:
```suggestion
export function requiresWrite(_: Request, res: Response, next: NextFunction) {
```

`readOnlyMode` is a bit weird - makes it seem like the endpoint is allowed in read-only

Reply:
How about `writeEnabled` ?

Reply:
hmm ok :)